src/transformers/cache_utils.py

```diff
@@ -184,6 +187,120 @@ def from_legacy_cache(cls, past_key_values: Optional[Tuple[Tuple[torch.FloatTens
         return cache
 
 
+class QuantCache(Cache):
+    def __init__(self, n_bits: int = 2, q_group_size: int = 64, residual_length: int = 128) -> None:
```

Comment:
missing: `__init__` docstring, including a reference to the KIVI paper (and a brief explanation of the quantization strategy)